client/src/App.js

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+import logo from './logo.svg';
+
+
+function App() {
+  return (
+    <div className="App">
```

Comment:
Let's remove this div and use a fragment instead :)

You might want to remove all the create-react-app code to start clean while you are in there as well